src/test/java/com/google/api/codegen/testdata/discoveries/csharp/csharp_taskqueue.v1beta2.json.baseline

```diff
@@ -7,39 +7,31 @@
 // 2. Install the C# client library by adding a dependency on the relevant NuGet
 //    package. Libraries published by Google are owned by google-apis-packages:
 //    https://www.nuget.org/profiles/google-apis-packages
-
 using Google.Apis.Auth.OAuth2;
```

Comment:
IIRC, in some languages we have been leaving a blank line after the header comment; we should probably be consistent unless there are known differences in idiom for particular languages.